erts/emulator/beam/erl_driver.h

```diff
@@ -289,10 +281,7 @@ typedef struct erl_drv_entry {
 			 unsigned int *flags); /* Works mostly like 'control',
 						  a synchronous
 						  call into the driver. */
-    void (*event)(ErlDrvData drv_data, ErlDrvEvent event,
-		  ErlDrvEventData event_data);
-                                /* Called when an event selected by 
-				   driver_event() has occurred */
+    void *unused_event_callback;
```

Comment:
Nit: if your aim is to maintain binary compatibility then make this entry "void (*unused_event_callback)(void);", i.e. keep it a function pointer.  C does not guarantee that function pointers and void* have the same size.